src/objects/core/admin.py

```diff
@@ -13,6 +18,32 @@ class ObjectTypeAdmin(admin.ModelAdmin):
     )
     readonly_fields = ("_name",)
 
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path(
+                "<int:objecttype_id>/versions/",
+                self.admin_site.admin_view(self.versions_view),
+            )
+        ]
+        return my_urls + urls
+
+    def versions_view(self, request, objecttype_id):
+        versions = {
+            "count": 0,
+            "next": None,
+            "previous": None,
+            "results": [],
+        }
+        if objecttype := self.get_object(request, objecttype_id):
+            client = build_client(objecttype.service)
+            try:
+                response = client.get(objecttype.versions_url)
+                versions = response.json()
```

Comment:
This doesn't handle pagination currently, I think it's best to do instantiate `versions = []` and then in the code below use the `pagination_helper` from `zgw-consumers` to return all the results: https://zgw-consumers.readthedocs.io/en/latest/recipes.html#handling-pagination-in-api-endpoints